internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -2409,6 +2397,25 @@ func CleanupLegacyStaticPodFiles(runtime.Sequence, any) (runtime.TaskExecutionFu
 	}, "cleanupLegacyStaticPodFiles"
 }
 
+// ReloadMeta reloads META partition after disk mount, installer run, etc.
+func ReloadMeta(runtime.Sequence, any) (runtime.TaskExecutionFunc, string) {
+	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) error {
+		err := r.State().Machine().Meta().Reload(ctx)
+		if os.IsNotExist(err) {
+			err = nil
+		}
+
+		return err
```

Comment:
```suggestion
		if !os.IsNotExist(err) {
			return err
		}

		return nil
```



seems easier to read

Reply:
reworked similar to this